python-sdk/docs/astro/sql/operators/load_file.rst

```diff
@@ -256,5 +256,12 @@ Users can also load data from an HTTP API:
 
 Default Datasets
 ~~~~~~~~~~~~~~~~
-* Input dataset - Source file for the operator.
-* Output dataset - Target table of the operator.
+If you are using the Astro Python SDK version 1.1 or later, you do not need to make any code
```

Comment:
@phanikumv  do we already explain or point to a doc that explains the datasets?

Reply:
yeah we can point to the Dataset explanation in https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/concepts.rst#datasets

Reply:
fixed